app/src/ui/branches/branches-container.tsx

```diff
@@ -67,11 +67,21 @@ export class BranchesContainer extends React.Component<
     }
   }
 
-  public render() {
-    const branchName = this.props.currentBranch
-      ? this.props.currentBranch.name
-      : this.props.defaultBranch || 'master'
+  private getBranchName = (): string => {
+    const { currentBranch, defaultBranch } = this.props
+    if (currentBranch != null) {
+      return currentBranch.name
+    }
+
+    if (defaultBranch != null) {
+      return defaultBranch.name
+    }
 
+    return 'master'
```

Comment:
i know this is from the existing code, but should this be defaulting to `master`? it seems like `current branch` might be more accurate? if you agree i can file a separate issue for this.